common/pkg/timezone/timezone_freebsd.go

```diff
@@ -0,0 +1,7 @@
+//go:build !windows && !linux
+
+package timezone
+
+// O_PATH value on freebsd. We must define O_PATH ourselves
+// until https://github.com/golang/go/issues/54355 is fixed.
+const O_PATH = 0x00400000 //nolint:staticcheck // ST1003: should not use ALL_CAPS
```

Comment:
Absolutely non-blocking, c/common doesn’t have a stable API, so this ultimately makes no difference: Make this package-private `o_Path` or `oPath`, so that it is not a part of the public API of this subpackage?

Reply:
actually we have redefined the exact same openDirectory() function in libpod.

So I think maybe it makes sense to put this into a util pkg instead and then import it in podman